nervjs-v1.2.5-keyed/src/Store.es6.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+function _random(max) {
+    return Math.round(Math.random()*1000)%max;
+}
+
+export class Store {
+    constructor() {
+        this.data = [];
+        this.selected = undefined;
+        this.id = 1;
+	}
+    buildData(count = 1000) {
+        var adjectives = ["pretty", "large", "big", "small", "tall", "short", "long", "handsome", "plain", "quaint", "clean", "elegant", "easy", "angry", "crazy", "helpful", "mushy", "odd", "unsightly", "adorable", "important", "inexpensive", "cheap", "expensive", "fancy"];
+        var colours = ["red", "yellow", "blue", "green", "pink", "brown", "purple", "brown", "white", "black", "orange"];
+        var nouns = ["table", "chair", "house", "bbq", "desk", "car", "pony", "cookie", "sandwich", "burger", "pizza", "mouse", "keyboard"];
+        var data = [];
+        for (var i = 0; i < count; i++)
+            data.push({id: this.id++, label: adjectives[_random(adjectives.length)] + " " + colours[_random(colours.length)] + " " + nouns[_random(nouns.length)] });
+        return data;
+    }
+    updateData(mod = 10) {
+        for (let i=0;i<this.data.length;i+=10) {
+        	this.data[i] = Object.assign({}, this.data[i], {label: this.data[i].label + ' !!!'});
+        }
+    }
+    delete(id) {
+        var idx = this.data.findIndex(d => d.id === id);
+        this.data.splice(idx, 1);
+    }
+    run() {
+        this.data = this.buildData();
+        this.selected = undefined;
+    }
+    add() {
+        this.data = this.data.concat(this.buildData(1000));
+    }
+    update() {
+        this.updateData();
+    }
+    select(id) {
+        this.selected = id;
+    }
+    runLots() {
+        this.data = this.buildData(10000);
+        this.selected = undefined;
+    }
+    clear() {
+        this.data = [];
+        this.selected = undefined;
+    }
+    swapRows() {
+    	if(this.data.length > 10) {
+    		var a = this.data[4];
```

Comment:
These should be `1` and `998` like in [v1.2.4/utils.es6.js](https://github.com/krausest/js-framework-benchmark/blob/a49ca102504abc53ce3c9454479513cc381e5b53/nervjs-v1.2.4-keyed/src/utils.es6.js#L37-L45)

Reply:
Thanks for point this out. Is it ok now?

Reply:
Looks ok.